### PR TITLE
Make MathKeyboard IDisposable

### PR DESCRIPTION
**Pull request by charlesroddie**
fixes: https://github.com/verybadcat/CSharpMath/issues/178

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
Comment (Happypig375) [inline NuGet.Config:4]: Is this file important?

**Comment by charlesroddie** [comment]
VS adds a nuget.config file automatically. Is this OK or does something need fixing @Happypig375 ?

File: NuGet.Config (hunk 1 above)
Comment (charlesroddie) [reply NuGet.Config:4]: VS adds a nuget.config file automatically. Is this OK or does something need fixing @Happypig375 ?

**Comment by Happypig375** [comment]
My Visual Studio does not do this. I am confused as to why does this happen.

File: NuGet.Config (hunk 1 above)
Comment (charlesroddie) [reply NuGet.Config]: I've removed it from this PR so we figure out what to do later when it crops up again.

**Review by FoggyFinder — APPROVED** [review]

**Comment by charlesroddie** [comment]
@Happypig375 could you queue a nuget release with this please?

**Pull request merged**